packages/client/src/editor/components/Stickies.tsx

```diff
@@ -7,69 +7,66 @@ import { Notes } from '@material-ui/icons';
 const UUID_NAMESPACE = '940beed9-f057-4088-a714-a9f5f2fc6052';
 
 interface prop {
-    get: (id: UUID) => string;
-    save: (content: string, id: UUID) => void;
-    remove: (id: UUID) => void;
+  get: (id: UUID) => string;
+  save: (content: string, id: UUID) => void;
+  remove: (id: UUID) => void;
 }
 
 export const notes: Note[] = [];
 
 export const addNote = (pos: Coordinate) => {
-    notes.push({
-      UUID: v5('line' + (notes.length-1).toString(), UUID_NAMESPACE),
-      position: pos,
-      text: 'empty note'
-    });
-    console.log('added note');
+  notes.push({
+    UUID: v5('line' + (notes.length - 1).toString(), UUID_NAMESPACE),
+    position: pos,
+    text: ''
+  });
+  console.log('added note');
 };
 
 const moveNote = (id: UUID, newPos: Coordinate) => {
-    let index = notes.findIndex( note => note.UUID == id);
-    if (index != -1) {
-      notes[index].position = newPos;
-    }
+  let index = notes.findIndex(note => note.UUID == id);
+  if (index != -1) {
+    notes[index].position = newPos;
+  }
 };
-  
+
 const updateNote = (content: string, id: UUID) => {
-    let index = notes.findIndex( note => note.UUID == id);
-    if (index != -1) {
-      notes[index].text = content;
-      console.log(content);
-    }
+  console.log('updating note', { id, content });
+  let index = notes.findIndex(note => note.UUID == id);
+  if (index != -1) {
+    notes[index].text = content;
+    console.log('updated note', { id, content });
+  }
 };
-  
+
 const deleteNote = (id: UUID) => {
-    let index = notes.findIndex( note => note.UUID == id);
-    if (index != -1) {
-      notes.splice(index,1);
-    }
+  let index = notes.findIndex(note => note.UUID == id);
+  if (index != -1) {
+    notes.splice(index, 1);
+  }
 };
 
 const getNote = (id: UUID) => {
-    let index = notes.findIndex( note => note.UUID == id);
-    if (index != -1) {
-      return notes[index].text;
-    } else return 'Note with ${id} not found'
+  let index = notes.findIndex(note => note.UUID == id);
+  if (index != -1) {
+    return notes[index].text;
+  } else return 'Note with ${id} not found';
 };
 
 const Stickies = () => {
-    const [notelist, addSticky] = useState(notes);
-    const listItems = notelist.map( note => {
-        <div key={note.UUID}>
-            {new StickyNote({
-                id: note.UUID,
-                get: getNote,
-                save: updateNote,
-                delete: deleteNote
-            })}
-        </div>
-    });
+  console.log({ notes });
+  const listItems = notes.map(note => (
```

Comment:
I've changed it to referncing the global `notes` directly. This is by no means proper React -- the component still has no way of knowing when something was added to `notes` and should be rendered. It somehow works now, but I'm not even sure why.

In proper React, everything that a component (e.g. Stickies) displays should be based on either:
1. its props
2. its local state (that is, coming from useState). This state can only be updated by calls to the function returned by useState (in the removed code above, that was `addSticky`, by convention it should be `setNoteList`)
3. some global values, but supplied in an appropriate manner. That is, not referencing a global varaiable directly, but using [context](https://reactjs.org/docs/context.html) or a global state management library, such as Redux or [Recoil](https://recoiljs.org/). We happen to already use Recoil for storing the editr/cursor mode and I think we should use it more.